app/controllers/manageBusinesses/add/AddIncomeSourceStartDateController.scala

```diff
@@ -204,8 +204,8 @@ class AddIncomeSourceStartDateController @Inject()(val authorisedFunctions: Auth
                          isChange: Boolean): String = {
 
     ((isAgent, isChange, incomeSourceType) match {
-      case (false, false, SelfEmployment) => routes.AddBusinessNameController.show()
-      case (_, false, SelfEmployment) => routes.AddBusinessNameController.showAgent()
+      case (false, false, SelfEmployment) => routes.AddBusinessNameController.show(isAgent = false, isChange = false)
+      case (_, false, SelfEmployment) => routes.AddBusinessNameController.show(isAgent = true, isChange = false)
```

Comment:
Is there a reason that these don't have the manageBusinesses bit in routes?

Reply:
Ok, and if you command click the .show method on these, does it take you to the AddIncomeSourceController in the manageBusinesses folder? Just want to make sure we are going to the correct place in the new journey

Reply:
it redirects to controllers/manageBusinesses/add/ReverseRoutes.scala